cccatalog-api/cccatalog/api/models.py

```diff
@@ -88,6 +89,10 @@ class Image(OpenLedgerModel):
 
     tags = JSONField(blank=True, null=True)
 
+    tags_list = ArrayField(
+        models.CharField(max_length=255), blank=True, null=True
+    )
```

Comment:
Our sample data depends on this field, so I added it back. Otherwise our migrations wouldn't work. It isn't really related to this PR.

Reply:
Why does the sample data depend on this field/why can't it be updated to use the new field?

Reply:
`tags_list` is the legacy format for storing tag information; now we have a fancier JSON-based format stored in `tags`. It was a bit of an oversimplification for me to say that our only dependency on this field is in our sample data; it's also used upstream in the CC Catalog, so removing it without breaking the ingestion process would take some coordination and database administration. Rather than spending a few hours dealing with removing the field and potentially risking disruption of ingestion, it's easier to just live with it and ignore it